explainer.md

```diff
@@ -186,225 +181,236 @@ https://user-images.githubusercontent.com/93594/141100217-ba1fa157-cd79-4a9d-b3b
 
 When the transition is complete, the transition elements created by the UA are removed, revealing the real Page-B.
 
-# The MPA API
+# The cross-document transition API
 
-There are a lot of open questions around the API, but this section will give a flavor of the direction.
+## Page-A
 
-Little thought has been giving to naming in these APIs, which will be improved when there's stronger consensus around the concepts.
+Page-A must offer elements to use for a transition, otherwise no transition will happen.
 
-## Creating a transition element
+### Via CSS
 
-### CSS properties
+Page-A can offer an element to be used in a transition via CSS, using the `page-transition-tag` property:
 
 ```css
+:root {
+  page-transition-tag: root;
+}
 .header {
   page-transition-tag: header;
-  page-transition-capture: container-and-child;
 }
 ```
 
-Setting a `page-transition-tag` marks this element as a transition element, and the `page-transition-capture` property is used to indicate how it's captured (as a single image vs as a computed style + content image).
+The tag is a [`<custom-ident>`](https://www.w3.org/TR/css-values/#identifier-value). It's recommended to use 'root' to refer to the bottommost element that covers the viewport, which is usually `:root`, but this isn't enforced.
 
-Multiple elements could have the same `page-transition-tag` value, which is useful for aggregate content such as user comments.
+Multiple elements can share a tag. For example, each comment of an article can use `page-transition-tag: comment`, and each comment will be offered for the transition.
 
-Elements on Page-A and Page-B which have the same `page-transition-tag` are automatically associated with each other. If multiple elements have the same value, they're associated in DOM order.
+The other modes mentioned in this document, such as "computed style + content image", and "retaining hierarchy" will be exposed via other CSS properties, and `page-transition` will be used as a shorthand. However, this is out of scope for 'v1'.
 
-Using CSS properties means values can change depending on viewport (`@media`) and browser support (`@supports`).
+### Via JavaScript
 
-### JS API
+The JavaScript API extends on the capabilities of the CSS API. In particular, it allows the developer to react to the URL of Page-B.
```

Comment:
It's not just about reacting to the URL of page-B. It's also about the different states page-A can be in when the transition occurs. For example a transition after page-A's initial load vs after the user has interacted with page-A which might have changed its content.